Gemfile.lock

```diff
@@ -160,6 +160,8 @@ GEM
       mini_mime (>= 0.1.1)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
+    meta-tags (2.13.0)
```

Comment:
REVIEW THE USE IF META TAGS TO MAKE USE OF ITS SEO FXN AND TO MAKE SURE YOU DONT REPEAT YOURSELF MUCH